docs/fsharp/tutorials/getting-started/getting-started-vscode.md

```diff
@@ -1,4 +1,278 @@
+---
+title: Getting Started with F# in Visual Studio Code with Ionide
+description: Learn how to write and run F# code in Visual Studio Code with Ionide 
+keywords: visual f#, f#, functional programming, .NET, Visual Studio Code, vscode, Ionide
+author: cartermp
+manager: wpickett
+ms.date: 09/28/2016
+ms.topic: article
+ms.prod: .net-core
+ms.technology: .net-core-technologies
+ms.devlang: devlang-fsharp
+ms.assetid: 49775139-082e-442f-b5a2-dd402399b5d2
+---
+
 # Getting Started with F# in Visual Studio Code with Ionide
```

Comment:
it'd be a nice touch to add a mini-TOC up top linking to the anchor tags within this document like
```
[Getting Started with F# in Visual Studio Code with Ionide](#getting-started-with-f-in-visual-studio-code-with-ionide)

[Installing Visual Studio Code and the Ionide plugin](#installing-visual-studio-code-and-the-ionide-plugin)
```

especially if you choose to include the troubleshooting section I mention below, so the reader knows it's there before they encounter any issues.